diesel/src/pg/query_builder/only.rs

```diff
@@ -0,0 +1,96 @@
+use crate::{
+    expression::Expression,
+    expression::ValidGrouping,
+    pg::Pg,
+    query_builder::{AsQuery, AstPass, FromClause, QueryFragment, QueryId, SelectStatement},
+    query_source::QuerySource,
+    result::QueryResult,
+    JoinTo, SelectableExpression, Table,
+};
```

Comment:
Diesel does use unnested use statements everywhere else. I would prefer keeping it that way.